w3af_console

```diff
@@ -75,16 +78,19 @@ def _usage():
     print(USAGE_DOC)
 
 
-def _generate_run_commands(script_file, profile, force_profile):
+def _generate_run_commands(script_file, profile, force_profile, target_url):
     """
     Given the user configuration, generate the commands to run in the console
     user interface at startup.
     
     :param script_file: The script (-s) file name
     :param profile: The profile (-p) name
     :param force_profile: Should we run the profile? (-P)
+    :target_url: Target URL for scan
     """
     commands_to_run = []
+    if target_url:
+        commands_to_run.extend(['target', 'set target {}'.format(target_url), 'back'])
```

Comment:
Please validate `target_url` using a white-list to make sure it only contains `0-9a-zA-z:/@-.?&=` (and maybe some other chars that are valid in URLs and I'm forgetting).

Do this to prevent "w3af console command injection" when someone specifies a `target_url` containing new lines / carriage returns.